src/proxy-animation.js

```diff
@@ -1553,6 +1651,10 @@ export function animate(keyframes, options) {
 
   if (timeline instanceof ScrollTimeline) {
     animation.pause();
+    if (timeline instanceof ViewTimeline) {
+      details = proxyAnimations.get(proxyAnimation);
```

Comment:
The `details` variable is not declared in this function. This will throw a `ReferenceError` in strict mode.

Reply:
Fixed.